src/app/src/test/kotlin/org/vaccineimpact/api/tests/logic/BurdenEstimateLogicTests.kt

```diff
@@ -141,6 +142,87 @@ class BurdenEstimateLogicTests : MontaguTests()
         verify(repo).getEstimateWriter(defaultEstimateSet)
     }
 
+    @Test
+    fun `gets burden estimate set data`()
+    {
```

Comment:
Can we test the other bits of logic - that an invalid id combo throws an error, that stochastic sets are unsupported.

Reply:
Yep, putting those checks into the database tests